create_civet_image.sh

```
@@ -680,10 +731,14 @@ if [[  ${#_arg_colourmap[@]} == 2 ]]; then
 
   convert ${tmpdir}/colourbar_pos.png ${tmpdir}/colourbar_neg.png -append ${tmpdir}/colourbar.png
 
+  colourbar_label_offset=$(calc "${threshold_low_left}/${threshold_high_left}*(585-290)" )
+
   if [[ -n ${_arg_colourbar_labels} ]]; then
-    colourbar_label_low=(-gravity SouthWest -annotate +1400+290 "±$(echo ${_arg_colourbar_labels} | cut -d, -f 1)")
-    colourbar_label_high=(-gravity SouthWest -annotate +1400+580 "+$(echo ${_arg_colourbar_labels} | cut -d, -f 2)")
-    colourbar_label_high+=(-gravity SouthWest -annotate +1400+12 "−$(echo ${_arg_colourbar_labels} | cut -d, -f 2)")
+    colourbar_label_low=(-gravity SouthWest -annotate +1400+$(calc 290+${colourbar_label_offset}) "+$(echo ${_arg_colourbar_labels} | cut -d, -f 1)")
```

Shellcheck (0.10.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)